app/Jobs/ApplicationDeploymentJob.php

```
@@ -580,16 +580,14 @@ private function deploy_docker_compose_buildpack()
             }
         } else {
             $composeFile = $this->application->parse(pull_request_id: $this->pull_request_id, preview_id: data_get($this->preview, 'id'));
-            $this->generate_runtime_environment_variables();
-            if (filled($this->env_filename)) {
-                $services = collect(data_get($composeFile, 'services', []));
-                $services = $services->map(function ($service, $name) {
-                    $service['env_file'] = [$this->env_filename];
+            // Always add .env file to services
+            $services = collect(data_get($composeFile, 'services', []));
+            $services = $services->map(function ($service, $name) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$name'. (undefined)

(UnusedFormalParameter)